brainspell/user_account_helpers.py

```diff
@@ -2,9 +2,11 @@
 
 import hashlib
 
-from torngithub import json_decode, json_encode
+from torngithub import json_decode
 
 from models import *
+# TODO: Updates to this file rely on removing json_encoding from the
```

Comment:
What does this mean? (i.e., why did we remove json_encode?)

Reply:
We're using the Postgres JSON type rather than the text type for these fields, so there's no need to encode before loading it into the database now.